src/components/Login/Login.js

```diff
@@ -10,19 +10,31 @@ class Login extends Component {
       email: '',
       password: ''
     };
+    this.setState({error: ''});
```

Comment:
Do not call setState in constructor, instead initialize the state like the email and password has been initialized above